src/main/java/com/bobocode/dto/User.java

```diff
@@ -0,0 +1,15 @@
+package com.bobocode.dto;
+
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+import org.springframework.hateoas.RepresentationModel;
+
+import java.util.List;
+
+@Data
+public class User {
+    private String id;
+    private String firstName;
+    private String lastName;
+    private List<Note> notes;
```

Comment:
IMO it's a good practice to initialize collections in entity class, not to catch NPE somewhere in the code
but it's up to you)

Reply:
Thanks, I'll fix it

Reply:
BTW, I really catched NPE, so I needed to cover it in business logic
<img width="790" alt="image" src="https://github.com/golichenko02/user-notes-api/assets/67119483/af9df025-7d26-4c8b-a5e5-d4d3f7f2fbfe">

Reply:
Fixed